modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm

```diff
@@ -88,6 +88,7 @@
 	greyscale_config_worn_digi = /datum/greyscale_config/jean_skirt/worn/digi
 	greyscale_colors = "#787878#723E0E#4D7EAC"
 	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = GROIN
```

Comment:
this specific part should be done modularly, or, upstream

Reply:
https://github.com/Skyrat-SS13/Skyrat-tg/pull/29598